.github/workflows/build.yml

```diff
@@ -3,16 +3,29 @@ name: CI
 on:
   push:
     branches: [main]
+  pull_request:
+    branches: [main]
   release:
     types: [published]
 
 jobs:
-  docker-release:
+  build-publish:
     uses: mekomsolutions/shared-github-workflow/.github/workflows/docker-build-publish.yml@main
     with:
+      context: senaite-without-sso/
       download-artifacts: false
       image-name: "senaite"
       image-version: "dev"
+    secrets:
+      DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_REGISTRY_USERNAME }}
+      DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_REGISTRY_PASSWORD }}
+  build-publish-sso:
+    uses: mekomsolutions/shared-github-workflow/.github/workflows/docker-build-publish.yml@main
+    with:
+      context: senaite-with-sso/
+      download-artifacts: false
```

Comment:
This is redundant as it defaults to `false`

Reply:
Done